topic-operator/src/test/java/io/strimzi/operator/topic/TopicOperatorBaseIT.java

```diff
@@ -566,17 +567,17 @@ protected void waitForEvent(KafkaTopic kafkaTopic, String expectedMessage, Topic
             List<Event> items = kubeClient.events().inNamespace(NAMESPACE).withLabels(labels.labels()).list().getItems();
             List<Event> filtered = items.stream().
                     filter(evt -> !preExistingEvents.contains(evt.getMetadata().getUid())
-                            && "KafkaTopic".equals(evt.getInvolvedObject().getKind())
-                            && kafkaTopic.getMetadata().getName().equals(evt.getInvolvedObject().getName())).
+                            && "KafkaTopic".equals(evt.getRegarding().getKind())
+                            && kafkaTopic.getMetadata().getName().equals(evt.getRegarding().getName())).
                     collect(Collectors.toList());
-            LOGGER.debug("Waiting for events: {}", filtered.stream().map(evt -> evt.getMessage()).collect(Collectors.toList()));
+            LOGGER.debug("Waiting for events: {}", filtered.stream().map(evt -> evt.getReason()).collect(Collectors.toList()));
             return filtered.stream().anyMatch(event ->
-                    Pattern.matches(expectedMessage, event.getMessage()) &&
+                    Pattern.matches(expectedMessage, event.getReason()) &&
                         Objects.equals(expectedType.name, event.getType()) &&
-                        event.getInvolvedObject() != null &&
-                        event.getLastTimestamp() != null &&
-                        Objects.equals("KafkaTopic", event.getInvolvedObject().getKind()) &&
-                        Objects.equals(kafkaTopic.getMetadata().getName(), event.getInvolvedObject().getName()));
+                        event.getRegarding() != null &&
+                        event.getDeprecatedLastTimestamp() != null &&
```

Comment:
If it's deprecated should we still be using it?

Reply:
I am not sure how important it is in TO. If you think it is not needed anymore, I can remove it.